src/Microsoft.Data.SqlClient/netcore/src/Common/perf/PerfRunner/PerfRunner.csproj

```
@@ -13,7 +13,7 @@
   <ItemGroup>
     <Compile Include="PerfRunner.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' != 'net46'">
+  <ItemGroup Condition="'$(TargetGroup)' != 'net461'">
```

I don't think there's any TargetGroup in this project, it's never used too.
Could you change this to:
```suggestion
  <ItemGroup Condition="'$(TargetFramework)' != 'net461'">
```

---

done.